daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/GlobalKey.scala

```diff
@@ -27,9 +27,6 @@ final class GlobalKey private (
 
 object GlobalKey {
 
-  def apply(templateId: Ref.TypeConName, key: Value): GlobalKey =
```

Comment:
Oh, we are using that one!

Reply:
Just replace it by `GlobalKey.assertBuild `